README.md

```diff
@@ -35,6 +35,15 @@ This project is in alpha status.
 
 See currently open [issues](https://github.com/Bee-Lab/bowerphp/issues).
 
+Contributing
+------------
+
+All contribution are welcome, just take a look at our [issues](https://github.com/Bee-Lab/bowerphp/issues) tracker if you wanna start somewher
```

Comment:
I am not an English purist (English is not even my first language), but I appreciate something like "want to" instead of "wanna". And, of course, "somewher*e*"